src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```diff
@@ -23,11 +23,7 @@
 /*! The constructor for the CenterOfBrightness module. It also sets some default values at its creation.  */
 CenterOfBrightness::CenterOfBrightness()
 {
```

Comment:
:small_red_triangle_down: Again I'd squash this commit.

Reply:
Done